_posts/2021-05-25-rust-and-tell.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Rust and Tell"
+date: "2021-05-25 19:00:00 +0200"
+categories: meetup
+location: online
+links:
+    "Meetup.com": https://www.meetup.com/Rust-Berlin/events/xxx
+talks:
+- title: "5 Traits for Beginners to Learn"
+  speaker:
+    name: "Tim McNamara"
+- title: "Ergonomic error handling in Rust"
+  speaker:
+    name: "Senyo Simpson"
+- title: "Building a cryptocurrency derivative exchange with Rust"
+  speaker:
+    name: "Konstantin Wünscher & Blas Rodriguez"
+---
+
+We are happy 🥳 🎉 to announce that the "Rust and Tell" online meetup for May will be on Tuesday, 25th!.
+
+**SPEAKERS WANTED 🙏**
+
+We desperatly need speakers for the following months; as we have exhausted our backlog of talks. Rust & Tell needs your help to keep going by proposing a talk. Send a message to [@evanxg852000](https://twitter.com/evanxg852000) or an email to [contact@berline.rs](mailto:contact@berline.rs).
```

Comment:
IMO we shouldn't be "desperate".
Something more like: "We're now looking for more speakers for the following months. Have something to talk about, a new project you're exploring or a new topic you just learned? ..."

What do you think?

Reply:
```
We're now looking for more speakers for the following months. Have something to talk about, a new project you're exploring or a new topic you just learned? The Rust & Tell community will be delighted to hear from you. Send a message to [@evanxg852000](https://twitter.com/evanxg852000) or an email to [contact@berline.rs](mailto:contact@berline.rs).
```

Reply:
Just pushed a change, thanks for the quick feedback